botCommands/simpleCommands.js

```diff
@@ -52,16 +52,17 @@ const data = {
       .setDescription(`
 Instead of asking if anyone can help you, ask your question outright so people can help you!
 
-**Bad**: "Hey, does anyone know how to set CSS styles with Javascript?"
+**Bad**:
+"Hey, does anyone know how to set CSS styles with Javascript?"
 
 **Good**:
-"Hey, I'm having trouble setting CSS styles via Javascript.
+"Hey, I'm having trouble setting CSS styles via Javascript."
```

Comment:
I don't believe this closing double quote goes here as it is meant to encapsulate the entire message for the "Good" example.

Reply:
Of course, I see that now.